contracts/wawaka/common/Response.cpp

```diff
@@ -24,74 +24,62 @@
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 Response::Response(void)
 {
-    status_ = false;
-    state_changed_ = false;
-    result_ = NULL;
+    value_ = json_value_init_object();
+    expected_value_type_ = (value_ == NULL ? JSONError : json_value_get_type(value_));
+
+    ww::value::Boolean status(true);
+    ww::value::Boolean response(true);
+    ww::value::Boolean state_changed(false);
+    ww::value::Array dependencies;
```

Comment:
regarding the  state_changed field. In the InvocationResponse, the default seems to be true for state_changed, here it seems to be false.

Reply:
doesn't matter.